woocommerce/woocommerce-gutenberg-products-block
# Implement `@wordpress/babel-preset-default` (drops IE11 support)

Taken from https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4362/files

This switches from `@babel/preset-env` to `@wordpress/babel-preset-default`. As a result of this we'll see smaller build sizes, at the expense of dropping IE11 support. This adopts what the latest `@wordpress/browserlist-config` settings are.

We may be able to remove more of the `babel-plugin`s, but this is not my expertise so I'm leaving them in place.

### Testing

This requires smoke testing, and build testing. 

- Delete `node_modules`
- Run `npm install`
- Run `npm run build`
- Confirm build was successful and smoke test blocks

### Performance Impact

Smaller files sizes due to the removal of IE11 support.

### Changelog

> Remove IE11 support in frontend builds.
